vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -262,7 +261,7 @@ void BatPublishers::saveVisitInternal(
              verified_old) {
     publisher_info->visits += 1;
     publisher_info->duration += duration;
-    publisher_info->score += concaveScore(duration);
+    publisher_info->score += concaveScore(duration * 1000);
```

Comment:
why is this `* 1000` we should specify the units `duration_in_seconds` or `duration_in_milliseconds` or whatever it is

Reply:
done